tests/integration/targets/dcnm_res_manager/tests/dcnm/dcnm_res_manager_merge.yaml

```diff
@@ -0,0 +1,257 @@
+##############################################
+##               SETUP                      ##
+##############################################
+
+- name: Remove local log file
+  local_action: command rm -f res_mgr.log
+
+- name: Delete Resources
+  cisco.dcnm.dcnm_res_manager: &rm_delete
+    state: deleted                                 # choose form [merged, deleted, query]
+    fabric: "{{ ansible_it_fabric }}" 
+    config:
+      - entity_name: "l3_vni_fabric"               # A unique name to identify the resource
```

Comment:
Please add a testcase to delete with just the `entity_name` if this is supported.

Reply:
Not supported. all parameters are mandatory